admin/src/components/dev-dialog.tsx

```diff
@@ -84,6 +84,11 @@ function CodeTab({ collection }: Props) {
     useForm<ExportOptionsValues>()
   const [status, setStatus] = React.useState<Status>('idle')
 
+  const { name, schema } = collection
+
+  Object.assign(schema, { created_at: { required: true, type: 'string' } })
+  Object.assign(schema, { updated_at: { required: true, type: 'string' } })
```

Comment:
Doing this is risky as it updates the original schema which is used to set up the schema from in the Edit tab